packages/at_invitation_flutter/lib/widgets/otp_dialog.dart

```diff
@@ -38,7 +38,7 @@ class _OTPDialogState extends State<OTPDialog> {
   @override
   Widget build(BuildContext context) {
     SizeConfig().init(context);
-    var deviceTextFactor = MediaQuery.of(context).textScaleFactor;
+    var deviceTextFactor = MediaQuery.of(context).textScaler.scale(20) / 20;
```

Comment:
what's the purpose of this change?

Reply:
textScaleFactor is deprecated, so updated that with textScaler